Makefile.am

```diff
@@ -135,6 +138,18 @@ check-unit:
 .PHONY: lint
 lint: gitignore execcmd black flake8 pylint
 
+.PHONY: venv
+venv:
+	python3 -m venv $(VENV_DIR) && \
+	$(VENV)/python3 -m pip install --upgrade pip && \
+	$(VENV)/python3 -m pip install -r docker/requirements.txt
+
+.PHONY: clean-venv
+clean-venv:
+	if [ -d "$(VENV_DIR)" ]; then \
+		rm -r $(VENV_DIR); \
+	fi
```

Comment:
Do we really need this? I never cleaned my ~/.venv/vdsm directory. Why would you need to recreate it?